packages/metastream-app/src/components/lobby/Disconnect.tsx

```diff
@@ -39,6 +40,7 @@ class _Disconnect extends Component<IProps> {
         <Link to="/">
           <MenuButton size="medium">{t('ok')}</MenuButton>
         </Link>
+        <MenuButton size="medium" onClick={reconnect}>{t('retry')}</MenuButton>
```

Comment:
The root element, with `styles.container`, is using flexbox and aligning all children into a column. I think if you wrap the two buttons in a `<div>`, it will switch to a horizontal layout.